pkg/services/grafana-apiserver/auth/authorizer/provider.go

```diff
@@ -17,12 +15,7 @@ func ProvideAuthorizer(
 	stackIDAuthorizer *stack.StackIDAuthorizer,
 	cfg *setting.Cfg,
 ) authorizer.Authorizer {
-	authorizers := []authorizer.Authorizer{
-		// This will allow privileged uses to do anything.
-		// In development mode, a privileged user is configured and saved into:
-		// ${data}/grafana-apiserver/grafana.kubeconfig
-		authorizerfactory.NewPrivilegedGroups(user.SystemPrivilegedGroup),
```

Comment:
🎉  -- this means we won't have the weird issues that system users can write to *any* namespaces even if totally invalid